pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: viam_sdk
 description: Viam Robotics Flutter SDK. Connect to your Viam robots using WebRTC or gRPC, and control your robots remotely.
-version: 0.12.0
+version: 0.12.1
```

Comment:
This file gets updated automatically when we do a new release, I don't think we should be modifying this value by hand.